addons/sale_management/models/sale_order_line.py

```diff
@@ -15,13 +15,16 @@ def _compute_name(self):
         # Take the description on the order template if the product is present in it
         super()._compute_name()
         for line in self:
-            if line.product_id and line.order_id.sale_order_template_id:
+            if line.product_id and line.order_id.sale_order_template_id and self._take_template_name():
                 for template_line in line.order_id.sale_order_template_id.sale_order_template_line_ids:
                     if line.product_id == template_line.product_id:
                         lang = line.order_id.partner_id.lang
                         line.name = template_line.with_context(lang=lang).name + line.with_context(lang=lang)._get_sale_order_line_multiline_description_variants()
                         break
 
+    def _take_template_name(self):
```

Comment:
Would add a small docstring, something like:

""" Allows overriding to avoid using the template lines descriptions for the sale order lines descriptions.
This is typically useful for "configured" products, for example when configuring an event ticket, as
the so line description content is then completed automatically with the selected event and ticket type. """